### DISCUSSION TITLE
Local Screens Not Working with 20210115 Update

### DISCUSSION THREAD

**Discussion by feebb** [rern/rAudio#569] [Fixed Issues]
Tried new 64-bit image with 2 systems (RPi4 with the Official 7" Screen and an RPi3B with an Adafruit 2.8" capacitive screen) and couldn't get the screen to work with either one. I've had no trouble with either screen using the previous 32-bit versions.
Am I doing something wrong?
Thanks!

**Comment by ceejayemm** [comment]
@feebb 

I have the new 64bit i20211015 image working with my RPi 3B+ and the official 7in screen without any further problems other than those previously reported in #426 where the right hand side of the screen does not allow/register touch operations.  All other functions work correctly. I have found previously that if the ribbon cable from the RPI to the screen is only slightly out of position then I get a black screen

Chris

**Reply by feebb** [reply]
I tried 64bit i20211015 image again with a different RPi4 and Official 7" Screen. On initial boot, got command line boot entries, and then just a blinking cursor. Through remote interface, I turned on Browser on RPi and got blotched colors and no image. On power-off, I saw the +R logo for a second, but on reboot, just got a black screen. 

No change with 20211019 update.

Same RPi4 and screen worked fine with rAudio-1-RPi4-20210911 image.

**Comment by feebb** [comment]
I've now tried the 64bit image with a RPi3B and still no local screen. The only thing visible on the display is the +R logo on power off. The "Browser on RPi" feature does not accept any changes, which makes me think that it does not recognize the screen. The screen worked fine with a 32-bit image from 20210924.
Is there something that must be done to make a 64-bit image work right on either a Pi3 or Pi4, such as update the firmware, etc.

**Comment by ceejayemm** [comment]
I have to agree with @feebb. I updated to the latest i20211010 image and applied the UPDATE branch to check on another issue.  The system appears to boot and is initially usable from my Windows 10 PC Pro via MS Edge but after a little while become unusable as if the system is continually rebooting - but i can't prove this.  The local screen on the RPI3B+ flashes a lot more that previously and displays the RAudio logo repeatedly.  I did notice that the UPDATE branch downloaded far fewer files than normal.  Has the UPDATE branch corrupted the main installation ?

Chris

**Comment by ceejayemm** [comment]
@feebb 

Just reflashed my card with latest i20211020 update which is telling me it has reverted back to the Chromium browser and still didn't have access to my local screen.  Then I remembered something I had to do in the past when restoring a backup (which I do as a matter of course with a new image) .  This worked for me to restore access to the local browser:

cd ~/.config/chromium
mv SingletonCookie CM_SingletonCookie
mv SingletonLock CM_SingletonLock			
mv SingletonSocket CM_SingletonSocket
systemctl restart localbrowser
systemctl status localbrowser

Chris

**Reply by rern** [reply]
Chris,

This is a good finding. 

@feebb,

Try: 
```sh
rm -rf /root/.config/chromium
systemctl restart localbrowser
```

**Reply by feebb** [reply]
@rern, @ceejayemm, 

Flashed the latest 20211020 image and Official Screen worked! It shows chromium is back and localbrowser.service is up and running. However, I did get the following error messages from systemctl status localbrowser:

`Oct 21 14:20:11 rAudio sudo[924]: [924:932:1021/142011.875290:ERROR:socket_posix.cc(96)] CreatePlatformSocket() failed: Address family not supported by protocol (97)
Oct 21 14:20:11 rAudio sudo[924]: [924:932:1021/142011.898348:ERROR:socket_posix.cc(96)] CreatePlatformSocket() failed: Address family not supported by protocol (97)
Oct 21 14:20:11 rAudio sudo[924]: [924:932:1021/142011.905540:ERROR:socket_posix.cc(96)] CreatePlatformSocket() failed: Address family not supported by protocol (97)
Oct 21 14:20:11 rAudio sudo[924]: [924:932:1021/142011.909717:ERROR:socket_posix.cc(96)] CreatePlatformSocket() failed: Address family not supported by protocol (97)
Oct 21 14:20:11 rAudio sudo[924]: [924:932:1021/142011.913764:ERROR:socket_posix.cc(96)] CreatePlatformSocket() failed: Address family not supported by protocol (97)
Oct 21 14:20:12 rAudio sudo[924]: [924:932:1021/142012.366230:ERROR:socket_posix.cc(96)] CreatePlatformSocket() failed: Address family not supported by protocol (97)
Oct 21 14:20:18 rAudio sudo[883]: [883:902:1021/142018.032104:ERROR:chrome_browser_main_extra_parts_metrics.cc(228)] crbug.com/1216328: Checking Bluetooth availability started. Please report if there is no report that this ends.
Oct 21 14:20:18 rAudio sudo[883]: [883:902:1021/142018.032385:ERROR:chrome_browser_main_extra_parts_metrics.cc(231)] crbug.com/1216328: Checking Bluetooth availability ended.
Oct 21 14:20:18 rAudio sudo[883]: [883:902:1021/142018.032413:ERROR:chrome_browser_main_extra_parts_metrics.cc(234)] crbug.com/1216328: Checking default browser status started. Please report if there is no report that this ends.
Oct 21 14:20:18 rAudio sudo[883]: [883:902:1021/142018.384536:ERROR:chrome_browser_main_extra_parts_metrics.cc(238)] crbug.com/1216328: Checking default browser status ended.`

I don't know if these errors mean anything.

When I tried the first 64bit image, updated through addons, to 20211020, systemctl status said unit localbrowser.service not found. And the chromium folder wasn't there to begin with.

Will now try the new image with other screens.

Thanks very much for your help!
